pubs/content.py

```diff
@@ -70,9 +70,14 @@ def read_text_file(filepath, fail=True):
     try:
         with _open(filepath, 'r') as f:
             content = f.read()
+            try:  # Python 2
```

Comment:
We could move this one to the `_open` function so that all python 2 stuff is handled there. It would return a `bytestring` only when `'b'` is in the `mode` argument, regardless of python version.